modules/wizard/view-wizard-page.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+/**
+ * Displays the wizard
+ *
+ * @since 2.7
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Don't access directly.
+};
+$admin_body_class[] = 'pll-wizard';
+$admin_body_class[] = 'wp-core-ui';
```

Comment:
The array should be defined before adding elements.  Here:
`$admin_body_class = array( 'pll-wizard', 'wp-core-ui' );`
Note sure why PHPCS doesn't warn here.
See for ex: 3b9a648b9254ca937d452303b8379ebd3e4ae59b

Reply:
Corrected